org.civicrm.bootstrapcivihr/scss/pages/vacancies.scss

```diff
@@ -0,0 +1,315 @@
+      margin-left: 20px;
+
+      .crm-i {
+        display: none;
+      }
+    }
+  }
+
+  .CRM_HRRecruitment_Form_HRVacancy {
+    .crm-block {
+      padding-top: 20px;
+
+      > table.form-layout-compressed,
+      > fieldset {
+        margin: 20px;
+      }
+
+      > table.form-layout-compressed {
+        margin-top: 0;
+
+        td {
+          &.label {
+            text-align: left !important;
+          }
+
+          > .select2-container,
+          > .crm-form-text,
+          > .crm-form-date-wrapper .crm-form-text:nth-child(2) {
+            min-width: 200px !important;
+          }
+        }
+      }
+    }
+
+    @media (max-width: 1340px) {
+      #s2id_permission_1 {
+        min-width: 200px !important;
+      }
+
+      .crm-form-multiselect {
+        min-width: 230px;
+        width: 230px;
+      }
+    }
+
+    @media (max-width: 1100px) and (min-width: 861px) {
+      .crm-form-multiselect {
+        min-width: 160px;
+        width: 160px;
+      }
+    }
+  }
+
+  #addMorePermission {
+    background: $brand-primary;
+    border-color: $brand-primary;
+    border-radius: $panel-border-radius;
+    color: $crm-white;
+    font-family: $font-family-base;
+    padding: 8px 12px;
+    text-shadow: none;
+    text-transform: uppercase;
+
+    &:hover {
+      background: $crm-blue-light;
+    }
+
+    > .add-icon {
+      background-image: none;
+      text-indent: 0;
+      margin-right: 3px;
+
+      &:after {
+        @include fa-icon($font-size-base, '\f067',$crm-white);
+        position: relative;
+        top: -1px;
+      }
+    }
+  }
+
+  table.vacancy-popup {
+    margin-bottom: 20px;
+  }
+
+  .view-hrvacancies {
+    .description {
+      margin-bottom: 20px;
+    }
+
+    td > a {
+      color: $brand-primary;
+    }
+  }
+
+  .CRM_HRRecruitment_Form_Search .crm-search-form-block {
+    margin-bottom: 20px;
+  }
+
+  #searchForm {
+    padding: 20px;
+
+    td {
+      padding: 0;
+    }
+
+    tr:nth-child(2) td {
+      padding: 20px 0;
+    }
+
+    #_qf_Search_refresh {
+      margin-bottom: 0 !important;
+    }
+
+    .listing-box {
+      border-color: $crm-copy;
+
+      > div {
+        background-color: transparent;
+
+        label {
+          color: $crm-copy;
+          font-weight: normal !important;
+          padding: 5px;
+        }
+      }
+    }
+
+    .font-size12pt {
+      font-size: $font-size-small;
+    }
+
+    .crm-form-text {
+      width: 312px;
+    }
+  }
+
+  &.page-civicrm-vacancy-dashboard .ui-dialog {
+    padding: 0;
+
+    &-content {
+      padding: 0;
+
+      .crm-block {
+        box-shadow: none;
+      }
+    }
+
+    table tr {
+      border-bottom: 0;
+
+      td {
+        padding-left: 0;
+        border-bottom: 0;
+      }
+    }
+
+    .advmultiselect tr:first-child td:nth-child(2) {
+      vertical-align: top;
+
+      input[type='button']:first-child {
+        margin-left: 0 !important;
+        margin-bottom: 15px !important;
```

Comment:
I see many !important in this PR. is it all necessary? Can u try to reduce the !important as much as you can?